### ISSUE TITLE
Restoring a session opens all previously open buffers in a single overlapping buffer

### ISSUE THREAD

**Issue by charludo** [NvChad/NvChad#1681]
**Describe the bug**
Restoring a session appears to only be opening a single buffer containing the last-editet file. Closing this buffer reveals another buffer, closing this one revelas the previous one, and so on. This also happens with a fresh install / no custom config

**To Reproduce**
1. remove custom config
2. PackerSync to uninstall plugins
3. open nvim, open two arbitrary files in two buffers
5. create a session with `mksession ~/Session.vim`
6. exit nvim
7. reopen nvim
8. `source ~/Session.vim`
9. only the last-opened buffer is displayed.
10. it is also not possible to tab-switch between them
11. close the buffer
12. the other buffer is revelaed
13. exit nvim and reopen again
14. `source ~/Session.vim`, same behaviour
15. run `PackerCompile`
16. interestingly, now both buffers show up

**Expected behavior**
sourcing a session file opens all previous buffers next to each other in the status bar

**Desktop (please complete the following information):**
 - Arch linux
 - alacritty
 - 0.8.0

**Comment by siduck** [comment]
and what happens when you have tabufline not lazyloaded?

**Comment by charludo** [comment]
I have this in my `chadrc`:
```lua
["NvChad/ui"] = {
      tabufline = {
          lazyload = false,
          overriden_modules = nil,
      },
},
```
which, I think, should disable lazy loading? From the [documentation](https://nvchad.com/config/nvchad_ui#statusline--tabufline) it sounds like lazy means the tabufline won't be shown if there's only a single buffer open. But it does show the tabufline for me, just with only one buffer.

Looks like this is related to / a duplicate of #1277. I see there's a couple of related commits, which I guessed fixed the issue (it was indeed working correctly for me until updating NvChad yesterday), but I guess something broke again, either here or in a recent nvim version?

**Comment by charludo** [comment]
Yes. This works for me as well. The issue only appears when sourcing a session file:
[parallax-2022-11-15_12.05.37.webm](https://user-images.githubusercontent.com/47758554/201905266-f1994d0a-65e1-4097-a2bd-ecd70bd8af0b.webm)
